Horizon/Horizon/Sources/Features/Dashboard/View/DashboardView.swift

```diff
@@ -28,41 +28,73 @@ struct DashboardView: View {
     }
 
     var body: some View {
-        BaseHorizonScreen {
-            InstUI.BaseScreen(
-                state: viewModel.state,
-                config: .init(refreshable: true)
-            ) { proxy in
-                VStack(spacing: 0) {
-                    ForEach(viewModel.programs) { program in
-                        if program.currentModuleItem != nil, !program.upcomingModuleItems.isEmpty {
-                            VStack(alignment: .leading, spacing: 12) {
-                                Size24BoldTextDarkestTitle(title: program.name)
-                                    .padding(.top, 16)
-                                CertificateProgressBar(
-                                    maxWidth: proxy.size.width - 2 * 16,
-                                    progress: program.progress,
-                                    progressString: program.progressString
-                                )
-                                moduleView(program: program)
-                            }
-                            .padding(.horizontal, 16)
-                            .background(Color.backgroundLight)
+        InstUI.BaseScreen(
+            state: viewModel.state,
+            config: .init(refreshable: true)
+        ) { proxy in
+            VStack(spacing: 0) {
+                ForEach(viewModel.programs) { program in
+                    if program.currentModuleItem != nil, !program.upcomingModuleItems.isEmpty {
```

Comment:
The condition is ok for now, but will be better remove it to ViewModel

Reply:
Yes, this will most probably be part of the interactor when the backend is ready.